src/main/java/org/wso2/carbon/inbound/poll/dbeventlistener/DBEventPollingConsumer.java

```diff
@@ -272,6 +265,10 @@ private String getColumnValue(ResultSet rs, String columnName, int type) {
                 columnValue = rs.getTime(columnName) + "";
             } else if (type == Types.TIMESTAMP) {
                 columnValue = rs.getTimestamp(columnName) + "";
+            } else if (type == Types.CLOB){
```

Comment:
Formatting issue. Space should be there after closing bracket.
Should be corrected as "else if (type == Types.CLOB) {"

Reply:
fix with commit 77dba88e588a7ed6959f3ce479e96a3abb87ca38